openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -336,7 +368,8 @@ def create_storage_class(self, secret_name=None,
             is_arbiter_vol=is_arbiter_vol,
             arbiter_avg_file_size=arbiter_avg_file_size,
             heketi_zone_checking=heketi_zone_checking,
-            volumeoptions=volumeoptions, skip_cleanup=skip_cleanup)
+            volumeoptions=volumeoptions, skip_cleanup=skip_cleanup,
+            glusterfs_registry=False)
```

Comment:
Need to pass variable from function definition, not default value. This should be like  `glusterfs_registry=glusterfs_registry`

Reply:
done